test/pathSatisfies.js

```diff
@@ -0,0 +1,17 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+describe('pathSatisfies', function() {
+
+  var isPositive = function(n) { return n > 0; };
+
+  it('returns true if the specified object path satisfies the given predicate', function() {
+    eq(R.pathSatisfies(isPositive, ['x', 'y'], {x: {y: 1}}), true);
+  });
+
+  it('returns false otherwise', function() {
+    eq(R.pathSatisfies(isPositive, ['x', 'y'], {x: {y: 0}}), false);
+  });
+
```

Comment:
I think this could use another test demonstrating the behavior on

    R.pathSatisfies(isPositive, ['x', 'y'], {x: {z: 42}}